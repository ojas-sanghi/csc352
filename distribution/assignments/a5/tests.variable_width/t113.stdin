    uikls    en  it    kockk   qpfwbx  hnh  mlwvq 


   vyh    rekbhte fhwgw  vvmohil yclzxcux  gf    srnkj gvut   dnr uca  pwmo   ejj    
