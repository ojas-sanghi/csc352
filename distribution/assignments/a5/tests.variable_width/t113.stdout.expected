 uikls     hnh  vvmohil  dnr
    en   mlwvq yclzxcux  uca
    it     vyh       gf pwmo
 kockk rekbhte    srnkj  ejj
qpfwbx   fhwgw     gvut
